Cargo.toml

```diff
@@ -30,8 +30,8 @@ default = []
 backtraces = []
 
 [dependencies]
-cosmwasm-std = { git = "https://github.com/line/cosmwasm", tag = "v0.14.0-0.4.0", features = ["iterator","staking","stargate"] }
-cosmwasm-vm = {  git = "https://github.com/line/cosmwasm", tag = "v0.14.0-0.4.0", features = ["iterator","staking","stargate"] }
+cosmwasm-std = { git = "https://github.com/line/cosmwasm", rev = "50ed856dd95ebfa31dc2e495ffe3c2506ab25bb9", features = ["iterator","staking","stargate"] }
```

Comment:
It will be changed to  squash merged commit of PR https://github.com/line/cosmwasm/pull/153